README.md

```diff
@@ -2,55 +2,63 @@
 
 # Project Name
```

Comment:
- [OPTIONAL] Please kindly update your project name, replace the placeholder text with your project name as this will make your `readme file` more informative and professional.